components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json

```diff
@@ -196,6 +196,22 @@
       {
         "Name": "apim:admin_operations",
         "Roles": "admin"
+      },
+      {
+        "Name": "registry:view",
+        "Roles": "admin,Internal/registry_admin,Internal/registry_user,Internal/registry_browser"
+      },
+      {
+        "Name": "registry:write",
+        "Roles": "admin,Internal/registry_admin"
+      },
+      {
+        "Name": "registry:entry_view",
```

Comment:
The same roles mentioned in the previous comment should be applied here as well.

Reply:
fixed